bots/activity.bot.js

```diff
@@ -47,10 +54,13 @@ const tallyMessage = (msg)=>{
  * the specified `thresholdSeconds`. Return the surviving timestamps
  * in a new array.
  */
-const cullTimestamps = (timestamps, thresholdSeconds)=>{
+const cullTimestamps = (timestamps, thresholdSeconds, channelKey)=>{
 	const minimumTimestamp = unixNow() - thresholdSeconds;
 	const cutoffIndex = _.sortedIndex(timestamps, minimumTimestamp);
-	return _.slice(timestamps, cutoffIndex);
+	const culled = _.slice(timestamps, cutoffIndex);
+	debug(`[ActivityBot]: Culled <#${channelKey}> timestamps ${timestamps} -> ${culled}`);
+	}
```

Comment:
Looks like you left a dangling brace here after adding the debug function.

Reply:
[yep](https://github.com/stolksdorf/slack-higgins/commit/9c2e633db013f97bb8d48680cb351e29c5bf606a)